config.json

```diff
@@ -843,6 +843,22 @@
         "traits"
       ]
     },
+    {
+      "uuid": "1bce70ca-db1a-46c8-a314-07d3fda921c2",
+      "slug": "grep",
+      "core": false,
+      "unlocked_by": "poker",
+      "difficulty": 7,
+      "topics": [
+        "file_io",
+        "string_parsing",
+        "string_functions",
+        "utf8",
+        "conditionals",
```

Comment:
Just noticed that Travis failed, which is almost certainly because the topics are out of order and have inconsistent indentation. Just run `configlet fmt` and that should fix this. Travis will need to pass before I merge, of course.